$NetBSD: patch-ab,v 1.32 2015/04/08 05:35:39 wiz Exp $

Please note that this patch cannot be regenerated automatically. What would be
the first hunk has manually been split into two and the context reduced for the
second half because the managesieve patch changes the line
        # Protocols we want to be serving: imap imaps pop3 pop3s
into
        # Protocols we want to be serving: imap imaps pop3 pop3s managesieve
and the patch needs to apply to both versions. So this line must be kept out of
the context.

--- dovecot-example.conf.orig	2009-06-01 05:21:42.000000000 +0200
+++ dovecot-example.conf
@@ -12,9 +12,6 @@
 # Default values are shown for each setting, it's not required to uncomment
 # those. These are exceptions to this though: No sections (e.g. namespace {})
 # or plugin settings are added by default, they're listed only as examples.
-# Paths are also just examples with the real defaults being based on configure
-# options. The paths listed here are for configure --prefix=/usr
-# --sysconfdir=/etc --localstatedir=/var --with-ssldir=/etc/ssl
 
 # Base directory where to store runtime data.
 #base_dir = /var/run/dovecot/
@@ -23,3 +20,3 @@
 # If you only want to use dovecot-auth, you can set this to "none".
-#protocols = imap imaps
+protocols = imap pop3
 
@@ -86,7 +83,7 @@
 #ssl_listen =
 
 # SSL/TLS support: yes, no, required. <doc/wiki/SSL.txt>
-#ssl = yes
+#ssl = no
 
 # PEM encoded X.509 SSL/TLS certificate and private key. They're opened before
 # dropping root privileges, so keep the key file unreadable by anyone but
@@ -537,20 +534,20 @@
 
 protocol imap {
   # Login executable location.
-  #login_executable = /usr/libexec/dovecot/imap-login
+  #login_executable = @PREFIX@/libexec/dovecot/imap-login
 
   # IMAP executable location. Changing this allows you to execute other
   # binaries before the imap process is executed.
   #
   # This would write rawlogs into user's ~/dovecot.rawlog/, if it exists:
-  #   mail_executable = /usr/libexec/dovecot/rawlog /usr/libexec/dovecot/imap
+  #   mail_executable = @PREFIX@/libexec/dovecot/rawlog @PREFIX@/libexec/dovecot/imap
   # <doc/wiki/Debugging/Rawlog.txt>
   #
   # This would attach gdb into the imap process and write backtraces into
   # /tmp/gdbhelper.* files:
-  #   mail_executable = /usr/libexec/dovecot/gdbhelper /usr/libexec/dovecot/imap
+  #   mail_executable = @PREFIX@/libexec/dovecot/gdbhelper @PREFIX@/libexec/dovecot/imap
   #
-  #mail_executable = /usr/libexec/dovecot/imap
+  #mail_executable = @PREFIX@/libexec/dovecot/imap
 
   # Maximum IMAP command line length in bytes. Some clients generate very long
   # command lines with huge mailboxes, so you may need to raise this if you get
@@ -564,7 +561,7 @@ protocol imap {
   # Support for dynamically loadable plugins. mail_plugins is a space separated
   # list of plugins to load.
   #mail_plugins = 
-  #mail_plugin_dir = /usr/lib/dovecot/imap
+  #mail_plugin_dir = @PREFIX@/lib/dovecot/imap
 
   # IMAP logout format string:
   #  %i - total number of bytes read from client
@@ -614,11 +611,11 @@ protocol imap {
 
 protocol pop3 {
   # Login executable location.
-  #login_executable = /usr/libexec/dovecot/pop3-login
+  #login_executable = @PREFIX@/libexec/dovecot/pop3-login
 
   # POP3 executable location. See IMAP's mail_executable above for examples
   # how this could be changed.
-  #mail_executable = /usr/libexec/dovecot/pop3
+  #mail_executable = @PREFIX@/libexec/dovecot/pop3
 
   # Don't try to set mails non-recent or seen with POP3 sessions. This is
   # mostly intended to reduce disk I/O. With maildir it doesn't move files
@@ -678,7 +675,7 @@ protocol pop3 {
   # Support for dynamically loadable plugins. mail_plugins is a space separated
   # list of plugins to load.
   #mail_plugins = 
-  #mail_plugin_dir = /usr/lib/dovecot/pop3
+  #mail_plugin_dir = @PREFIX@/lib/dovecot/pop3
 
   # Workarounds for various client bugs:
   #   outlook-no-nuls:
@@ -706,7 +703,7 @@ protocol lda {
   # Support for dynamically loadable plugins. mail_plugins is a space separated
   # list of plugins to load.
   #mail_plugins = 
-  #mail_plugin_dir = /usr/lib/dovecot/lda
+  #mail_plugin_dir = @PREFIX@/lib/dovecot/lda
 
   # If user is over quota, return with temporary failure instead of
   # bouncing the mail.
@@ -720,7 +717,7 @@ protocol lda {
   #deliver_log_format = msgid=%m: %$
 
   # Binary to use for sending mails.
-  #sendmail_path = /usr/lib/sendmail
+  #sendmail_path = /usr/sbin/sendmail
 
   # Subject: header to use for rejection mails. You can use the same variables
   # as for rejection_reason below.
@@ -739,7 +736,7 @@ protocol lda {
 ##
 
 # Executable location
-#auth_executable = /usr/libexec/dovecot/dovecot-auth
+#auth_executable = @PREFIX@/libexec/dovecot/dovecot-auth
 
 # Set max. process size in megabytes.
 #auth_process_size = 256
@@ -869,7 +866,7 @@ auth default {
   # database (passwd usually), you can use static userdb.
   # REMEMBER: You'll need /etc/pam.d/dovecot file created for PAM
   # authentication to actually work. <doc/wiki/PasswordDatabase.PAM.txt>
-  passdb pam {
+  #passdb pam {
     # [session=yes] [setcred=yes] [failure_show_msg=yes] [max_requests=<n>]
     # [cache_key=<key>] [<service name>]
     #
@@ -902,15 +899,15 @@ auth default {
     #   args = session=yes %Ls
     #   args = cache_key=%u dovecot
     #args = dovecot
-  }
+  #}
 
   # System users (NSS, /etc/passwd, or similiar)
   # In many systems nowadays this uses Name Service Switch, which is
   # configured in /etc/nsswitch.conf. <doc/wiki/AuthDatabase.Passwd.txt>
-  #passdb passwd {
+  passdb passwd {
     # [blocking=yes] - See userdb passwd for explanation
     #args = 
-  #}
+  }
 
   # Shadow passwords for system users (NSS, /etc/shadow or similiar).
   # Deprecated by PAM nowadays.
